<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="PulseTable" name="pulseTableView"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QToolButton" name="addPulseButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources/resources.qrc">
           <normaloff>:/icons/plus.png</normaloff>:/icons/plus.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="removePulseButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources/resources.qrc">
           <normaloff>:/icons/minus.png</normaloff>:/icons/minus.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Period Duration</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ExponentialSpinBox" name="doubleSpinBox">
       <property name="maximum">
        <double>10000000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ExponentialSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>src.gui.widgets.exponential_spinbox</header>
  </customwidget>
  <customwidget>
   <class>PulseTable</class>
   <extends>QTableView</extends>
   <header>src.gui.widgets.tables.pulse_table</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
